ksp_plugin_adapter/burn_editor.cs

```diff
@@ -59,12 +59,9 @@ class BurnEditor : ScalingRenderer {
 
   // Renders the |BurnEditor|.  Returns true if and only if the settings were
   // changed.
-  public bool Render(string header,
-                     bool enabled,
-                     double? actual_final_time = null) {
+  public bool Render(string header, double final_time) {
```

Comment:
I think we should retain some sort of `enabled=false` on anomalous manoeuvres: editing them would happen without seeing any resulting effects.
A manoeuvre cannot be anomalous because of its own parameters, except for its timing; could it make sense to have the timing enabled on all manoeuvres, but the other parameters only enabled on non-anomalous ones?

Reply:
Yes, that makes sense.  It's also a good visual feedback.  Will do.